src/Controller/Component/FlashComponent.php

```diff
@@ -111,6 +112,14 @@ public function set($message, array $options = [])
             $messages = (array)$this->_session->read('Flash.' . $options['key']);
         }
 
+        if ($options['duplicate'] === false) {
```

Comment:
I would prefer using `if ($foo)`, `if (!$foo)` everywhere when `$foo` is know to be boolean.